<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>177</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>315</width>
    <height>177</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>315</width>
    <height>177</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="uniqvalues_cb">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>90</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Unique values</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>102</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioButtonGSM">
      <property name="text">
       <string>GSM</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="radioButtonLTE">
      <property name="text">
       <string>LTE</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="radioButtonUTMS">
      <property name="text">
       <string>UTMS</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="radioButtonALL">
      <property name="text">
       <string>ALL</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>207</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>host</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>4444</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>74</width>
     <height>157</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="mtsRB">
      <property name="text">
       <string>MTS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="TELE2RB">
      <property name="text">
       <string>TELE2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="BEELINERB">
      <property name="text">
       <string>BEELINE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="MEGAFONRB">
      <property name="text">
       <string>MEGAFON</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="SMARTRB">
      <property name="text">
       <string>SMART</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="HCCRB">
      <property name="text">
       <string>HCC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="ALLRB">
      <property name="text">
       <string>ALL</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
